coldfront/core/user/forms.py

```diff
@@ -164,6 +175,26 @@ class UserProfileUpdateForm(forms.Form):
         help_text='The number must be in E.164 format (e.g. +12125552368).',
         label='Phone Number', required=False)
 
+    def clean_phone_number(self):
+        cleaned_data = super().clean()
```

Comment:
`clean` is still being called in each of these methods. The cleaned data can instead be accessed via `self.cleaned_data` as in `clean_first_name` below.

(And again, no need to `pop`.)